### ISSUE TITLE
I WILL PAY FOR 1.16 SUPPORT

### ISSUE THREAD

**Issue by ModoSN** [amadornes/MCMultiPart#145]
@amadornes I will literally pay for you to add 1.16 support. We need cable covers!

**Comment by d0x7** [comment]
Yep... been playing a small-ish 1.16 modpack with a few friends recently and AE2 stuff is covered behind facades and then... then there's mekanism cables/pipes/tubes everywhere..

**Comment by amadornes** [comment]
Right. I should probably reply to this in here since we continued the conversation on Discord.

I'm currently working on my thesis and I have about a month and a half left until I'm done with university.  
After that I'm moving and building a new office office space, and once that's done I can sit down and look into it.

I'll need mod developers who are interested in supporting it to work with me on ironing out the bugs, though.  
I don't want a repeat of MCMP2 where people started dropping support for it because it was buggy, but never reported any bugs themselves.

**Comment by emkirsh** [comment]
So, ForgeMultipart is actually still being developed as CB Multipart, and has been updated to 1.16. I can't vouch for its stability, but the last commit was only a week ago, so it's safe to say the devs are active.
https://www.curseforge.com/minecraft/mc-mods/cb-multipart
https://github.com/TheCBProject/CBMultipart
